packages/components/tsconfig.esm.json

```diff
@@ -18,9 +18,7 @@
     "moduleResolution": "node"
   },
   "include": [
-    "src/**/*.ts",
-    "spec/**/*.ts",
-    "stories/**/*.ts"
```

Comment:
Adding a TypeScript compile error in either spec/stories isn't caught. Removing them they're not part of the build and the `correct` structure in `dist` is intact.